Tabloid/Controllers/PostController.cs

```diff
@@ -34,5 +37,29 @@ public IActionResult Get(int id)
             }
             return Ok(post);
         }
+
+        [HttpGet("GetPostsByUser/{id}")]
```

Comment:
I would prefer to not see verbs in the resource names in the URLs. you've already got an HTTP verb, so PostsByUser is sufficient (we'll talk more about this today in the talk about REST)